### PR TITLE
Issue #61 and #62

### PR DESCRIPTION
**Pull request by markmelvin**
This resolves issue #61 and #62.

### CODE REVIEW

File: cocotb_test/simulator.py [hunk 1]
```diff
@@ -66,6 +67,11 @@ def __init__(
         if self.module is None:
             self.module = os.path.splitext(os.path.split(run_filename)[-1])[0]
 
+        self.workdir = self.sim_dir
+        absworkdir = os.path.abspath(workdir)
```
Comment (themperek) [inline cocotb_test/simulator.py]: You have to check if `workdir` is None

Comment (markmelvin) [reply cocotb_test/simulator.py]: Woops!

**Comment by themperek** [comment]
Could you change `workdir` for `work_dir` so it is a bit more consistent?

It would be great to have some simple tests for this other way you may get regression easily. But I can marge without.

PS. I have closed and open to fix CI.

**Comment by themperek** [comment]
@markmelvin Thank you!

**Comment by markmelvin** [comment]
You're welcome!

**Pull request merged**